bitcoinology/history-of-bitcoin-in-gaming.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "History of Bitcoin in GAMING"
+transcript_by: masud-abdulkadir via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=CB7HM-Hqi4o
+tags: ["gaming"]
+speakers: ["Christian Moss","Audience"]
+categories: ["meetup"]
+date: 2022-03-27
+---
+
+
+
+## INTRO
+
+Christian: Before I start, it's probably a good idea to make sure you have a Lightning wallet downloaded because at the end, I have some games we can play and actually get some Bitcoin through the games. So if you haven't got a Lightning wallet installed, please do so. I recommend my company's wallet, Zebedee. So if you just type in Zebedee on my t-shirt into the Google Play or App Store, you should be able to download it. But today, yeah, today I'm gonna talk about the history of Bitcoin in gaming. Okay, cool. So I've been told that, like, people here are kind of somewhat Bitcoin familiar. Is anybody here who's completely new to Bitcoin? Don't be shy. But everybody kind of knows what it is, so I don't have to explain the basics. Yeah, so yeah, I thought I'd just give an interesting presentation on the history of Bitcoin in gaming, or the history of gaming with Bitcoin. Because at the moment, actually, blockchain in gaming is actually quite a buzzword. I went to a gaming conference just last week and there's like a blockchain track but there's really no mention of Bitcoin. But it actually all kind of started on Bitcoin with gaming and I was there in 2013 I started to make Bitcoin games. But yeah, I just want to give the context. I think it's interesting so people kind of know how it started, what kind of went wrong, and what's getting better now with Zebedee and Lightning. Yeah, so before I start, I just want to go 
+
+## Money in games
```

Comment:
This heading transition mid-sentence are jarring. Can we place the header in a better spot?

Reply:
Yeah can easily move em before the sentence starts so it's less jarring.